docker/PlatypusDockerContainer/config/ini/alllocal.ini

```diff
@@ -0,0 +1,16 @@
+[COMMON]
+useRoddyVersion=current
+
+[DIRECTORIES]
+configurationDirectories=/home/roddy/config/xml
+pluginDirectories=/home/roddy/binaries/Roddy/plugins_R2.4
+
+[COMMANDS]
+commandLogTruncate=80                       # Truncate logged commands to this length. If <= 0, then no truncation.
```

Comment:
Consider setting this to zero, such that the diagnostic messages **don't** get truncated.